009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,449 @@
+
+\subsubsection{Kindness}
+\label{subsubsec: Kindness}
+
+\Cref{fig: Fixed High Kindness} and \Cref{fig: Fixed Low Kindness} illustrate the results from simulations with agents with high kindness ($>$70) and low kindness ($<$30) respectively. All other traits are left as default.
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, fixed high kindness, T7Only, 2000days, 20food, 153deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with fixed high kindness (153 deaths).}
+    \label{fig: Fixed High Kindness}
+\end{figure}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, fixed low kindness, T7Only, 2000days, 20food, 70deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with fixed low kindness (70 deaths).}
+    \label{fig: Fixed Low Kindness}
+\end{figure}
+
+A high kindness score sees double the deaths of a low kindness score. A high kindness score does not achieve stability however a low kindness score allows for the tower to reach in less than 200. The most likely explanation for this is that agents with a high kindness are potentially too selfless. It is likely that they consume in small quantities when food is available and thus do not have sufficient health when food isn't available. Thus, making them more vulnerable.
+
+\newpage
+\subsubsection{Inter-team Comparison}
+The following table shows the total number of deaths for simulations of various combinations of agent teams. All simulations are run with a simulation period of 500 days, shuffle period of 7 days and 10 team 7 agents with 10 agents of one other team type. The table is further broken down into food available per agent (5, 10 and 15).
+\begin{center}
+\begin{tabular} { | m{5em} | m{5em} | m{5em} | m{5em} | m{5em} | }
+  \hline
+    \textbf{Agents} & \textbf{food per agent} & \textbf{total deaths} & \textbf{team 7 deaths} & \textbf{other deaths} \\
+  \hline
+    2 & 5 & 463 & 273 & 190 \\
+  \hline
+    2 & 10 & 149 & 80 & 49 \\
+  \hline
+    2 & 15 & 0 & 0 & 0 \\
+  \hline
+    3 & 5 & 347 & 183 & 164\\
+  \hline
+    3 & 10 & 73 & 53 & 20\\
+  \hline
+    3 & 15 & 0 & 0 & 0 \\
+  \hline
+    4 & 5 & 217 & 92 & 125\\
+  \hline
+    4 & 10 & 0 & 0 & 0 \\
+  \hline
+    4 & 15 & 0 & 0 & 0 \\
+  \hline
+    5 & 5 & 213 & 107 & 106 \\
+  \hline
+    5 & 10 & 5 & 5 & 0 \\
+  \hline
+    5 & 15 &  0 & 0 & 0 \\ 
+  \hline
+    6 & 5 & 459 & 136 & 323\\
+  \hline
+    6 & 10 & 273 & 48 & 225\\
+  \hline
+    6 & 15 & 126 & 24 & 102\\
+  \hline
+\end{tabular}
+\end{center}
+\caption{Comparison between agent teams and team 7.}
```

Comment:
I think there is a missing '\begin{table} for this caption here.

Reply:
Silly me, thanks